tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
```

Comment:
Line 1-1: Added a docstring to describe the module as specified in the style guide.
```suggestion
import os
import shutil
```